esp/src/src-react/components/Title.tsx

```diff
@@ -234,8 +233,9 @@ export const DevTitle: React.FunctionComponent<DevTitleProps> = ({
                         </Stack.Item>
                     }
                     <Stack.Item align="center">
-                        <DefaultButton href="#/log" title={nlsHPCC.ErrorWarnings} iconProps={{ iconName: log.length > 0 ? "RingerSolid" : "Ringer" }} className={btnStyles.errorsWarnings}>
-                            <span className={btnStyles.errorsWarningsCount}>{`(${log.length})`}</span>
+                        <DefaultButton href="#/log" title={nlsHPCC.ErrorWarnings} className={btnStyles.errorsWarnings}>
```

Comment:
We should keep the Bell Icon and either overlay the count on it or beside it where the parens used to be?